eras/shelley/impl/src/Cardano/Ledger/Shelley/HardForks.hs

```diff
@@ -65,3 +66,12 @@ missingScriptsSymmetricDifference ::
   pp ->
   Bool
 missingScriptsSymmetricDifference pp = pvMajor (getField @"_protocolVersion" pp) > 6
+
+-- | Starting with protocol version 7, the reward calculation no longer
+-- filters out unregistered stake addresses during the calculation.
```

Comment:
During the calculation, or at the start of the calculation? I don't think we've ever had anything filter _during_, since that would have a different effect in the pulsed and un-pulsed version.

Reply:
good catch! We are filtering out addresses during the computation depending on whether they were registered at the start of the calculation. I'll update the comment, thank you!